src/main.rs

```diff
@@ -268,6 +268,39 @@ fn new_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
 }
 
 
+/// push subcommand implementation
+///
+fn push_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
+    // note: "remote" is always present since it is a required parameter
+    let mut remote = try_or_1!(repo.find_remote(matches.value_of("remote").unwrap()));
+
+    // accumulate the refspecs to push
+    let refspecs : Vec<String> = if let Some(issues) = matches.values_of("issue") {
+        // push a specific list of issues
+        issues.map(Oid::from_str).abort_on_err()
+              .map(|issue| repo.get_issue_refs(issue)).abort_on_err()
+              .flat_map(git2::References::names).abort_on_err()
```

Comment:
Pls put the `abort_on_err()` call on the next line. Its much better for readability.